algebras/algebra/src/main/scala/endpoints/algebra/Requests.scala

```diff
@@ -37,13 +39,13 @@ trait Requests extends Urls with Methods with PartialInvariantFunctorSyntax with
   implicit def reqEntityInvFunctor: InvariantFunctor[RequestEntity]
 
   /**
-    * Empty request.
+    * Request without a body.
+    * Use `description` of [[endpoints.algebra.Endpoints#endpoint]] to document an empty body.
     */
-  //TODO rename to emptyBody and maybe make uniform with all other by adding ()
-  def emptyRequest: RequestEntity[Unit]
+  def emptyBody: RequestEntity[Unit]
```

Comment:
I’m not a fan of this change because this is inconsistent with the other constructors for `RequestEntity`, which all contain `Request` in their name (e.g. `textRequest`, `jsonRequest`, etc.).

Also, another problem is that we have to find a name that is unambiguous between requests and responses. An `emptyBody` could mean an embty request body as well as an empty response body.

Reply:
fair enough, I am going to close this pull request and open a new one for just the improved docs.